rules.ini

```diff
@@ -147,6 +147,7 @@ cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release|)\.(?:dll|exe|lib|
 Discord = (?:^|/)(?:lib)?discord(?:|-rpc|_game_sdk)\.(?:dll|dylib|so)$
 EpicOnlineServices = (?:^|/)(?:lib)?eossdk
 FMOD = (?:^|/)(?:lib)?fmod(?:l|ex|exl|studio|studiol|)(?:64)?\.(?:dylib|dll|so)$
+Intel_XeSS = xess\.dll$
```

Comment:
This will match anythingxes.dll, why?

is ig|lib not enough?

Reply:
fair, i was just lazy

Reply:
Add (?:^|/)

Reply:
knew i forgot something